### DISCUSSION TITLE
Issue with Accessing GitHub Copilot as a Verified Student

### DISCUSSION THREAD

**Discussion by black-zero358** [community/community#128010] [GitHub Education]
### Hi everyone,

Hi everyone,

I'm a verified student in the GitHub Education program. Recently, I tried to redeem the GitHub Copilot offer through the Global Campus benefits.

However, when I click the "Get access" button on the [signup page](https://github.com/github-copilot/free_signup), 

![image](https://github.com/community/community/assets/53086059/5093b9a4-6dad-41b1-bd68-f78f2c7dfdbb)

I get the error message: "Your account is unable to sign up for Copilot. Please contact Support."

![image](https://github.com/community/community/assets/53086059/b4183c83-17b2-4057-8002-368411946d84)

I'm not sure why this is happening, as I believe I meet all the eligibility requirements. I've contacted GitHub Support, but I'm also looking for advice from the community.

Has anyone else faced this issue? How did you resolve it?

Thanks!

**Comment by samus-aran** [comment]
Hi @black-zero358 

We can see you have been answered via your Support ticket. We'll go ahead and close this discussion for you. 👍
